mne_bids/path.py

```diff
@@ -120,22 +132,88 @@ def __str__(self):
                     key = 'rec'
                 basename.append('%s-%s' % (key, val))
 
-        if self.suffix is not None:
-            basename.append(self.suffix)
+        if self.kind is not None:
+            if self.extension is not None:
+                basename.append(f'{self.kind}{self.extension}')
+            else:
+                basename.append(self.kind)
 
         basename = '_'.join(basename)
-        if self.prefix is not None:
-            basename = op.join(self.prefix, basename)
-
         return basename
 
+    @property
+    def fpath(self):
```

Comment:
should this be documented in the Attributes of the class? Can you check what mne-python does for the Epochs object and its properties?

Reply:
I added it in similarly to Epochs. Lmk how it looks.